CIP-0911/README.md

```diff
@@ -0,0 +1,358 @@
+---
+CIP: CIP-0911?
+Title: Disaster Recovery Plan for Cardano
+Category: Cardano Information
```

Comment:
```suggestion
Category: 
```
Not sure what to suggest on this one

Reply:
@Ryun1 @kevinhammond it points those active in recovery efforts to a collection of tools, so I'd recommend `Tools` even though the CIP isn't a tool(s) specification.

Reply:
p.s. In this case the "tools" would also include the human elements in this proposal's application for either disaster recovery or preparedness (e.g. the subset of fallback block producers... which themselves include technical components).

Reply:
`Tools` is a green light as per today's CIP meeting ✅